samcli/lib/init/cli_template_modifier.py

```diff
@@ -0,0 +1,211 @@
+"""
+Classes used to parse and update template when tracing is enabled
+"""
+import logging
+from typing import Any, List
+from yaml.parser import ParserError
+from samcli.yamlhelper import parse_yaml_file
+
+
+LOG = logging.getLogger(__name__)
+
+
+class GlobalsSection:
+    def __init__(self):
+        self.globals = "Globals:\n"
+        self.resource = "Resources:\n"
+        self.function = "  Function:\n"
+        self.tracing = "    Tracing: Active\n"
+        self.comment = (
+            "# More info about Globals: "
+            "https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst\n"
+        )
+
+    def get_field(self, field: str):
+        """
+        Parameters
+        ----------
+        field : str
+            Global field value
+
+        Returns
+        -------
+        str
+            field and it value
+        """
+        return self.tracing if field == "Tracing" else None
+
+
+class TemplateModifier:
+    def __init__(self, location):
+        self.template_location = location
+        self.template = self._get_template()
+        self.copy_of_original_template = self.template
```

Comment:
Should we do a `deepcopy` here for the original template? In case it is been changed in the process, this will still write the changed template instead of the original one.